src/mds/query.py

```diff
@@ -66,9 +71,20 @@ async def search_metadata(
 
     def add_filter(query):
         for path, values in queries.items():
-            query = query.where(
-                db.or_(Metadata.data[list(path.split("."))].astext == v for v in values)
-            )
+            if "*" in values:
+                query = query.where(
+                    db.or_(
+                        Metadata.data[list(path.split("."))].astext != "",
+                        Metadata.data[list(path.split("."))].astext == "",
```

Comment:
Yeah i couldn't figure out a cleaner way to do it 🤔 
Because we are filtering JSON, we need to use `astext`, and then empty strings and missing values both get cast to `""`. So I am explicitly adding values matching `""`

Reply:
Maybe a comment makes sense, because `var == "" or var !=""` doesn't seem right in the first look. wdyt?

Reply:
What about:
```
path = list(path.split("."))
field = path.pop()
query = query.where(Metadata.data[path].has_key(field))
```

Reply:
thanks @johnfrancismccann 🦸